tests/components/pyload/snapshots/test_diagnostics.ambr

```diff
@@ -2,10 +2,8 @@
 # name: test_diagnostics
   dict({
     'config_entry_data': dict({
-      'host': '**REDACTED**',
       'password': '**REDACTED**',
-      'port': 8000,
-      'ssl': True,
+      'url': 'https://pyload.local:8000/prefix',
```

Comment:
Previously the host was redacted, now it isn't anymore?

These might be internal information right?

../Frenck